src/fragments/lib/logging/ios/setup_logging/20_initialize_with_config.mdx

```diff
@@ -0,0 +1,43 @@
+In your mobile app, create and add a `amplifyconfiguration_logging.json` to the same location as the `amplifyconfiguration.json` file.  Ensure that the file is included in the `Copy Bundle Resources` build phase.
```

Comment:
the copy bundle resources, is that something that a Swift developer would know? This is pure ignorance on my end.

Reply:
Yes, it's a build configuration/step in Xcode